lib/_app/AppBar/AppBar.tsx

```diff
@@ -1,63 +1,108 @@
-import {
-  AppBar as MuiAppBar,
-  Container,
-  IconButton,
-  Typography,
-} from '@mui/material';
-import React, { useState } from 'react';
 import Image from 'next/image';
-import { Icon } from '~/lib/components/Icon';
-import SearchIcon from '~/public/images/icons/search-regular.svg';
-import TimesIcon from '~/public/images/icons/times-regular.svg';
-import s from './AppBar.module.scss';
 import Link from 'next/link';
+import React, { useState } from 'react';
+import { useRouter } from 'next/router';
+import clsx from 'clsx';
 
 export const AppBar = () => {
   const [showSearch, setShowSearch] = useState(false);
+  const router = useRouter();
+  const isHomepage = router.pathname == '/';
 
   return (
-    <MuiAppBar
-      elevation={1}
-      color="transparent"
-      position="static"
-      className={s.AppBar}
-    >
-      <Container maxWidth="md" className={s.container}>
-        <div className={s.logo}>
-          <div style={{ display: showSearch ? 'block' : 'none' }}>
-            <div title="Google search results" className="gcse-search" />
-          </div>
-
-          <div style={{ display: showSearch ? 'none' : 'block' }}>
+    <nav className={isHomepage ? 'py-10' : 'py-4'}>
+      <section
+        className={clsx(
+          'px-4 container mx-auto flex gap-4 items-center justify-center w-full h-auto',
+          isHomepage ? 'flex-col' : 'flex-row',
```

Comment:
You can `dynamic classes` stored in variables. I think it's cleaner to have them added to the HTML elements like this instead. I used `clsx` to combine the string of classes with the conditional classes. I did that for all of the class variables you had.

Let me know if you have any concerns. If you don't have any concerns, you can mark this as resolved.

Reply:
I see, this is how you use it in this language, thank you sir!